apps/api/src/app/preferences/preferences.spec.ts

```diff
@@ -548,7 +548,7 @@ describe('Preferences', function () {
             },
           },
         },
-        type: PreferencesTypeEnum.SUBSCRIBER_GLOBAL,
+        type: PreferencesTypeEnum.USER_WORKFLOW,
```

Comment:
Fixing these tests after correcting the logic to ignore subscriber preferences when `readOnly: true`